apps/charterafrica/src/payload/blocks/EmbeddedDocuments.js

```diff
@@ -46,14 +46,13 @@ const EmbeddedDocuments = {
       type: "group",
       fields: [
         {
-          name: "group",
+          name: "groupId",
```

Comment:
What about the current code that's using group? Deploying this code as is would break that functionality.